cmake/OpenCVPCHSupport.cmake

```diff
@@ -60,11 +74,15 @@ MACRO(_PCH_GET_COMPILE_FLAGS _out_compile_flags)
         endif()
     ENDFOREACH(item)
 
-    GET_DIRECTORY_PROPERTY(_directory_flags DEFINITIONS)
```

Comment:
> Please, let me know if this file breaks your builds - I will revert parts not necessary for the fix.

No, it does not break anything :) it's just my interest. Thanks for the complete answer.